src/common/transformations/src/transformations/transpose_sinking/ts_general.cpp

```diff
@@ -58,6 +62,8 @@ TSGeneralBackward::TSGeneralBackward() {
     add_matcher<TSInterpolateBackward>();
     add_matcher<TSSliceBackward>();
     add_matcher<TSGatherBackward>();
+    add_matcher<TSCumSumBackward>();
+    add_matcher<TSTileBackward>();
```

Comment:
I suppose we should use `ADD_MATCHER` macro to enable CC in this umbrella transformation